ops/README.md

```diff
@@ -189,65 +189,46 @@ Each instance should have a responsible entity, which we currently model as a [
+### Dependencies
 
-### Additional dependencies
+In order to automatically set up a virtual machine:
 
-To test the changes without impacting the production server, a Vagrantfile is provided to test the changes locally in a virtual machine. 
-[Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [Docker](https://docs.docker.com/get-docker/) are therefore required.
+1. Install [Vagrant](https://www.vagrantup.com/docs/installation/).
+2. Install [VirtualBox](https://www.virtualbox.org/wiki/Downloads) to manage virtual machines. If you prefer Docker, or have an Apple Silicon machine, install [Docker](https://docs.docker.com/get-docker/) instead.
+3. Create a dedicated SSH key with no password: `ssh-keygen -f ~/.ssh/ota-vagrant -q -N ""`. This key will be automatically used by Vagrant.
 
-### SSH with Vagrant
+> VirtualBox is not compatible with Apple Silicon (M1…) processors. If you have such a machine, you will need to use the Docker provider. Since MongoDB cannot be installed on ARM, it is skipped in the infrastructure installation process. This means you cannot test the MongoDB storage adapter with Vagrant with an Apple Silicon processor.
 
-In order to deploy with Ansible in Vagrant, you need to create a custom SSH key with no password by using:
+### Launch
 
 ```
-ssh-keygen -f ~/.ssh/ota-vagrant -q -N ""
+vagrant up  # if you’re on an Apple Silicon processor or want to use Docker instead of VirtualBox, add ` --provider=docker`
 ```
 
-#### On a Mac with an Apple Silicon processor
+You can then deploy the code to the running machine with `ansible-playbook ops/site.yml` and all the options described above.
 
-VirtualBox is not compatible with Apple Silicon (M1…) processors. You will thus need to use the Docker provider.
+### Vagrant quick reference
 
-##### Setup
-
-Install Docker Desktop with the [official installer](https://docs.docker.com/get-docker/) or with `brew install docker`.
-
-##### Launch
-
-```
-vagrant up --provider=docker
-```
-
-You can then deploy the code to the running machine with:
+#### Connect to the virtual machine
 
 ```
-ansible-playbook ops/site.yml
+vagrant up
+vagrant ssh  # use "vagrant" as password
 ```
 
-:warning: Since [MongoDB cannot be installed on ARM](https://github.com/ambanum/OpenTermsArchive/issues/743), it is skipped in the infrastructure installation process. This means you cannot test MongoDB storage with Vagrant on ARM architecture.
-
-##### Connect to the running machine
+#### Start again with a clean virtual machine
 
 ```
-vagrant ssh
+vagrant halt  # stop machine
+vagrant destroy  # remove machine
+vagrant up
 ```
 
-and use `vagrant` as password
+#### Troubleshooting: Remote host identification has changed
 
-##### Stop and destroy
-
-```
-vagrant halt # stop machine
-vagrant destroy -f # remove machine
-```
-
-### TroubleShooting
```

Comment:
There is already a `Troubleshooting` section dedicated to vagrant earlier in the documentation, can you merge its contents with this section?